GDSCommon-Demo/GDSCommon-Demo/Mocks/MockInstructionWithImageViewModel.swift

```diff
@@ -0,0 +1,23 @@
+import GDSCommon
+import UIKit
+
+class MockInstructionWithImageViewModel: InstructionsWithImageViewModel {
+    var title: GDSCommon.GDSLocalisedString
```

Comment:
I don't think we need to specifically reference `GDSCommon` namespace on these types? I don't think it is _wrong_ just perhaps more wordy than it needs to be? What do you think? Same would apply for the `ButtonViewModel`

Reply:
Yeah it's a fair point. I had thought that it was a little more explanatory but when I looked again there was a lot 😅 going to remove. Thank you!